plugins/woocommerce/includes/tracks/class-wc-tracks.php

```diff
@@ -75,7 +75,7 @@ public static function get_server_details() {
 	 */
 	public static function get_role_details( $user ) {
 		return array(
-			'role'                   => ! empty( $user->roles ) ? $user->roles[0] : '',
+			'role'                   => ! empty( $user->roles ) ? reset( $user->roles ) : '',
```

Comment:
interesting usage of `reset()`!

Reply:
Could maybe also use `array_key_first` for readability 🤔

Reply:
Thanks for the suggestion! The `array_key_first` retrieves an array's first **key**, not the **first** value. So, it wouldn't work in this case.